README.md

```diff
@@ -400,6 +402,20 @@ Unlike record elements, product element defintions must include identifiers.
 (product int_pair first::int second::int)
 ```
 
+#### Imports
+
+It is possible to split type universe definitions among multiple files:
+
+```
+// root.ion:
+(import "a.ion")
+(import "b.ion")
+```
+
+The resulting type universe will contain all type domains from both `a.ion` and `b.ion`.  `root.ion` may also 
+define additional type domains.  The primary purpose of this is to be able to permute domains defined in another 
+file.
```

Comment:
I specifically chose to avoid the `include` term because that it is already used as part of the domain permutation feature.  The next logical one is `import`, IMHO.

Reply:
> Something to consider, should the import be more abstract? Instead of a file that implies paths, could one import a domain by name and leave the resolution to specific files as an exercise for the PIG implementation?

IMHO, no.  This isn't needed to accomplish the goal of being able to permute a domain defined in another file.  Also, it's hard for me to see a future where there are so many type domains this would provide a benefit that justifies the additional complexity.

> This can make sure you only pull in what you want and the granularity may be better modeled this way.

IMHO that kind of granularity isn't needed yet.

> This also leads to a question, how does namespacing work or not work with PIG? Is it one flat namespace?

PIG has no namespacing of its own outside of the type domain, and within a type domain it's only possible to reference types defined in the same type domain.  I'm not sure PIG will ever need anything more than it currently supports and what is proposed in #65.